documentation/images/images-default.md

```diff
@@ -0,0 +1,58 @@
+⚠️ This documentation file isn't complete ⚠️
+
+It is only there to serve as an example of how we'd document each template
+
+---
+
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File name | images-default.stencil |
+| Invocation example | `swiftgen storyboard -t default …` |
+| Language | Swift 3 |
+| Author | Olivier Halligon |
+
+## When to use it
+
+This template is the default template used by SwiftGen when parsing images in Asset Catalogs.
+It generates Swift 3 code and is suitable for most needs.
```

Comment:
Now that I think of it, when I thought about adding that section at first, is was more to say:

"This template is more suitable for when you name your images with underscores" or "This template is more suitable for when you use that "provides namespace" checkbox on the folders inside your xcasset catalog", and "this template supports multiple catalogs by creating a separate enum for each catalog", stuff like that